### ISSUE TITLE
libsim tests fail using dballe 8 (prerelease)

### ISSUE THREAD

**Issue by edigiacomo** [ARPA-SIMC/libsim#63]
I have packaged a pre-release of dballe and arkimet in simc/test and, before releasing them in the stable repository, I'd like to rebuild libsim against the new dballe to test our internal scripts of data provisions.

I tried to build the last version (6.2.8-3 ) but some tests fail (see https://copr.fedorainfracloud.org/coprs/simc/test/build/829830/):

```
PASS: vol7d_dballe_test
PASS: vol7d_dballe_test2
FAIL: dballe_test
PASS: dballe_test2
FAIL: dballe_test3
PASS: vol7d_dballe_test.sh
FAIL: dballe_test.sh
```

**Comment by edigiacomo** [comment]
I added the production blocking label because dballe#114 is labeled as production blocking too and the issue is fixed in the new dballe version (commit 20ae5b2a3dc33c6222ce9ded79ffc4ef8a558e78).

**Comment by dcesari** [comment]
Issue fixed in dballe
